code/game/objects/structures/kitchen_spike.dm

```diff
@@ -19,15 +21,24 @@
 	if(state != GRAB_KILL)
 		to_chat(user, SPAN_NOTICE("You need to grab \the [target] by the neck!"))
 		return FALSE
-	if(spike(target))
-		visible_message(SPAN_DANGER("[user] has forced [target] onto \the [src], killing them instantly!"))
-		for(var/obj/item/thing in target)
-			if(thing.is_equipped())
-				target.drop_from_inventory(thing)
-		qdel(target)
-		return TRUE
-	to_chat(user, SPAN_DANGER("They are too big for \the [src], try something smaller!"))
-	return FALSE
+	var/mob/living/carbon/human/H = target
+	var/list/damaged = H.get_damaged_organs(TRUE, FALSE)
+	for(var/obj/item/organ/external/chest/G in damaged)
+		if(G.brute_dam > 200)
+			to_chat(user, "[H] is too badly damaged to hold onto the meat spike.")
+			return 
+	visible_message(SPAN_DANGER("[user] is trying to force \the [target] onto \the [src]!"))
+	if(do_after(user, 80))
+		if(spike(target))
+			visible_message(SPAN_DANGER("[user] has forced [target] onto \the [src], killing them instantly!"))
+			target.damage_through_armor(201, BRUTE, BP_CHEST)
```

Comment:
Eight seconds with a meatspike = instakill, though I doubt anyone here would use this.

Reply:
Hilariously, it doesn't actually work at all.

Reply:
I'm pretty sure we didn't get all the code for it, or something is broken. At least it didn't work during testing. 

Which is part of why I didn't fool with it (plus there are eight more Prs around Eris' Guru fetish past this)